src/shared/components/case-editor/services/cases.service.ts

```diff
@@ -103,19 +102,19 @@ export class CasesService {
    *
    * EUI-2530 Dynamic Lists for Elements in a Complex Type
    *
-   * @param jsonResponse - {}
+   * @param jsonBody - {}
    */
-  private handleNestedDynamicLists(jsonResponse) {
+  private handleNestedDynamicLists(jsonBody: any): any {
```

Comment:
@paul-graham given that `jsonBody` is listed as `{}` in the params, would it work if you declared the same in the method parameter, i.e. `jsonBody: {}`, as opposed to just `any`?

Reply:
That won't work but I'll do one better as you're right - `any` is kinda pants.

Reply:
New version pushed with a better type than `any`. 🤦